<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinkedOutClass</class>
 <widget class="QMainWindow" name="LinkedOutClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LinkedOut</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>6</width>
     <height>0</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QFrame" name="MainFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(51,52,54);color:white;width:inherited;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="Signup_Frame">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>110</y>
       <width>380</width>
       <height>500</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(86, 152, 144);
border-style: solid; border-width: 2px; border-color: #000000;
	border-radius: 20px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="Username_Frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>380</width>
        <height>141</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:0;border-top:none;border-bottom:none;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="Username_Text_Input">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>291</width>
         <height>41</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid; border-width: 2px; border-color: #000000;</string>
       </property>
       <property name="maxLength">
        <number>9999</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>User name</string>
       </property>
      </widget>
      <widget class="QLabel" name="Username_Label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>User name: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="Signup_Button">
      <property name="geometry">
       <rect>
        <x>115</x>
        <y>440</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-style: solid; border-width: 2px; border-color: #000000;
	border-radius: 20px;
	background-color: rgb(86,152,144);
}


QPushButton:hover{
	background-color: rgb(100,180,160);
}</string>
      </property>
      <property name="text">
       <string>Sing up</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QFrame" name="Password_Frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>380</width>
        <height>141</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:0;border-top:none;border-bottom:none;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <widget class="QLineEdit" name="Password_Text_Input">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>291</width>
         <height>41</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid; border-width: 2px; border-color: #000000;</string>
       </property>
       <property name="maxLength">
        <number>9999</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLabel" name="Password_Label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Password:   </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QCheckBox" name="Remember_Me_Checkbox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string>Remember Me</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="Signup_Label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>80</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;font-weight:bold;font-size:20px;</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="Captcha_Frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>340</y>
        <width>380</width>
        <height>61</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:0;border-top:none;border-bottom:none;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <widget class="QLineEdit" name="Captcha_Text_Input">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>10</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid; border-width: 2px; border-color: #000000;</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="maxLength">
        <number>9999</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Captcha</string>
       </property>
      </widget>
      <widget class="QLabel" name="Captcha_Label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:0;border-top:none;border-bottom:none;font-style:italic;font-weight:bold;font-size:25px;border-right:none;
</string>
       </property>
       <property name="text">
        <string>Captcha: 5555</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QFrame" name="m_MessagesFrame">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>0</y>
       <width>300</width>
       <height>720</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(51,52,54,255);color:white;width:inherited;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>211</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="m_MessagesVerticalLayout">
       <item>
        <widget class="QFrame" name="m_MessageFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="m_MessageLabel">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>181</width>
            <height>81</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>sasdsa</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="m_MessagePushButton">
          <property name="geometry">
           <rect>
            <x>185</x>
            <y>0</y>
            <width>24</width>
            <height>24</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">

QPushButton {
	border:none;
}


QPushButton:hover{
		background-color:rgb(240,20,30);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="LinkedOut.qrc">
            <normaloff>:/LinkedOut/Resources/MessageCloseButtonOnHover.png</normaloff>:/LinkedOut/Resources/MessageCloseButtonOnHover.png</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LinkedOut.qrc"/>
 </resources>
 <connections/>
</ui>
